azure-pipelines.yml

```diff
@@ -172,6 +172,11 @@ jobs:
        build_scripts/windows/scripts/build.cmd
     displayName: 'Build Windows MSI'
 
+  - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
+    displayName: 'SBOM'
+    inputs:
+      BuildDropPath: 'build_scripts/windows/out/'
```

Comment:
Can we use `$(Build.ArtifactStagingDirectory)` for `BuildDropPath` to be consistent across all artifacts?

Reply:
Sorry, we can't.
Accroding to the [doc](https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/secure-supply-chain/ado-sbom-generator), we should use the same root path location that you publish your build drop to.
For MSI, our publish path is `build_scripts/windows/out/`.
If we want to make all the artifacts BuildDropPath same, we need change a lot of scripts and pipelines.
![image](https://user-images.githubusercontent.com/18628534/156723906-c77c3979-ad95-42a5-b1b7-d6d18bf87a09.png)